### PR TITLE
module one : Popup 

### PR DESCRIPTION
**Pull request by majdalkilany**
**Create a popup on the mobile version** 

**Create a popup on the desktop version**

### CODE REVIEW

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @majdalkilany, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Nice design.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js [hunk 1]
```diff
@@ -1,3 +1,72 @@
...
+    id: 1,
+    name: 'Multi-Post Stories Gain+Glory',
+    image: './assets/Img Plaholder.png',
+    description: `Lorem ipsum dolor sit amet consectetur, adipisicing elit. Cupiditate, sunt ea, saepe
+    aspernatur reiciendis doloribus deserunt perspiciatis eaque porro tempore consequatur voluptatibus nam veniam
+    officia, placeat praesentium dignissimos alias est?`,
+    technologies: ['Ruby on Rails', 'CSS', 'Javascript'],
+    liveVersion: 'https://majdalkilany.github.io/Portfolio-/',
+    sourceCode: 'https://github.com/majdalkilany/Portfolio-',
+  },
```
Comment (Shedrack-Sunday) [inline app.js:12]: - [ ] As suggested by the previous reviewer, kindly adjust the buttons to align properly to the left to in par with the paragraph. This is necessary to give a better user experience and shows attention to detail in your work 👍 
- You can consider adjusting using flexbox with a flex-start to make it fit 👍 

## View:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @majdalkilany   👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well so far in the project,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights   
- Good use of GitHub features ✔️   
- Beautiful and Nice UI 👍 

  
# Required Changes  🌵 

- [x] You made some impressive implementations which I must say is a job well done. However, there are some features who missed 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @majdalkilany , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done implementing the requested change. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**